components/automate-cli/cmd/chef-automate/certRotate_test.go

```diff
@@ -13,6 +13,9 @@ const (
 	RemoteFilePath = "198.51.100.0:/home/ec2-user/certs/public.pem"
 	LocalFilePath  = "/home/ec2-user/certs/public.pem"
 	ValidIP        = "198.51.100.0"
+	ValidIP1       = "198.51.100.1"
+	ValidIP2       = "198.51.100.2"
+	ValidIP3       = "198.51.100.3"
```

Comment:
This is not used in the code, please remove it.

Reply:
done